### DISCUSSION TITLE
楼盘烂尾 江西法院判由开发商还房贷

### DISCUSSION THREAD

**Discussion by HumorChen** [WeNeedHome/SummaryOfLoanSuspension#915] [General]
每经AI快讯，近日，江西省高安市人民法院审结了三起因开发商楼盘“烂尾”，购房人要求解除《商品房买卖合同》、《商品房按揭借款合同》的合同纠纷案件。

法院审理查明，三位购房人于2019年与高安市某公司签订了《商品房买卖合同》，购买该公司开发的楼盘房屋。购房人交纳了首付款后，购房人、开发商、银行三方签订了《商品房按揭借款合同》，向银行按揭借款支付购房余款，此后购房人按时归还银行贷款。但过了交房期开发商表示无法按时交房，且不能确定何时交房。

购房人遂依据合同约定向法院提起诉讼，请求判令解除《商品房买卖合同》、《商品房按揭借款合同》，由开发商返还首付款及已支付的银行贷款，后续贷款由开发商向银行归还。

法院审理认为，根据合同约定，开发商在最后期限的90天内无法交房，购房人有权要求解除合同。解除合同后开发商应将原告交付的首付款返还给购房人，另外购房人向银行支付的银行按揭贷款应由开发商返还给原告，未支付部分由开发商向银行支付。法院根据《中华人民共和国民法典》相关规定及《最高人民法院关于审理商品房买卖合同纠纷案件适用法律若干问题的解释》第十一条、第二十一条的规定，依法作出了上述判决。（中国法院网）

每日经济新闻

**Comment by k0spider** [comment]
这里我有个问题，开发商都已经是高负债了，拿什么钱来退给业主呢？

**Reply by wedxz** [reply]
所以。还是耍流氓，大鱼吃小鱼，小鱼吃虾米

**Reply by hrsgn** [reply]
政府和银行有责任，应该帮业主承担至少部分首付

**Reply by Lilithoffice365** [reply]
法院把包袱全给开发商了，开发商没钱去执行，只能走破产了

**Reply by k0spider** [reply]
> 法院把包袱全给开发商了，开发商没钱去执行，只能走破产了

破产的话开发商所有的负债就一笔勾销了，这还涉及到欠的上下游企业的债、欠的银行的债，欠的个人的债才是排在最后的。

**Reply by imhappyok** [reply]
你还想咋样啊，现在就这情况：
1.你继续还贷，房子大概率没有
2.1你不还了，首付看开发商情况，破产清算了有多的你就分点，没有就当首付打水漂，及时止损

**Reply by Wesleylr** [reply]
> 法院把包袱全给开发商了，开发商没钱去执行，只能走破产了

开发商破产那他底下的所有烂尾楼都不用业主还贷了，你觉得政府会让你这样做吗

**Reply by Siryedu** [reply]
> 所以。还是耍流氓，大鱼吃小鱼，小鱼吃虾米

解除购房合同，就不是第一债权人，拿不到首付款，受害的还是买房人

**Comment by toxin08x** [comment]
如果政府不接这个锅，甩锅给开发商的话，会不会导致开发商被挤兑破产，有些正在盖不会烂尾的房子也盖不完了呢？如果因此引发了连锁反应，那估计中国的房地产会遭受比预期还要更硬的着陆了吧...

**Reply by wedxz** [reply]
> 如果政府不接这个锅，甩锅给开发商的话，会不会导致开发商被挤兑破产，有些正在盖不会烂尾的房子也盖不完了呢？如果因此引发了连锁反应，那估计中国的房地产会遭受比预期还要更硬的着陆了吧...

那就摆烂吧

**Reply by TALK1949** [reply]
这不是正在进行时嘛，恒大都倒了，不然哪来那么多烂尾楼。

**Reply by Accelerator6666** [reply]
恒大妄图可以依仗自己规模实现自己大而不倒的目的，事实上政府已经没有足够财力来支持它大而不倒了。

**Reply by toxin08x** [reply]
还有就是开发商破产清算了，即使烂尾全责，没钱赔该烂那还是烂那。韭菜买了房子掏了首付和房贷，钱回不来还是打水漂。政府拿了钱的大头，监管缺失却一毛不拔当甩手掌柜。这烂尾楼问题其实还是无法解决：韭菜掏了钱没房子住，钱大概率回不来；开发商破产摆烂；政府吃人不吐骨头。

**Reply by sailing3** [reply]
政府接这个锅，那么意味着所有人为房地产买单，这个钱要从政府财政出意味着公共性的财政支出每个人平均下来都少了，所有人不管老的少的，有房的没房的全中国为这个买单。

**Comment by jingzz1** [comment]
能不还贷就算不错了，不过这判决，不知道银行会不会再起诉呢。

**Comment by ppinkiris** [comment]
最后政府不接盘还是兜不住，前提估计是把开发商的油水榨干再兜底，硬着陆呗，全社会的人为这些烂尾买单。

**Comment by xiejun946** [comment]
铁三角只有开发商接盘没用。政府和银行收的钱又不会吐出来，还是一点改变都没有

**Comment by Zcccczx** [comment]
停贷至少还有个烂尾楼，起诉开发商退房的话，房子、首付和几年交的房贷全无了

**Reply by tantalate** [reply]
有个烂尾楼也没办法住啊
起诉的早的人，趁着开发商账户里还有钱，说不定能把钱退回来。

**Comment by ineedabitchlikeu** [comment]
哈哈哈 开发商就是个狗 主人叫去咬人 就去咬 咬完就背锅 被做了狗肉火锅 笑死了

**Comment by lance111l** [comment]
停贷是经济问题 当然不是政治问题 开发商资金运作不行 倒闭了 美国倒闭的公司少吗
